### PR TITLE
Makes \s be properly used.

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Code that formerly used ternaries for pluralising counts will now use the built-in `\s` macro.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
!review

**Comment by MarinaGryphon** [comment]
Unsure if this counts as a bugfix or not, but nothing in this is player-facing so I don't think it needs a changelog.

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/cooking/recipes/recipes_chems.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/decl/recipe/chem/make_food(obj/container/C)
+	for(chem in resulting_reagents)
+		C.reagents.add_reagent(chem, resulting_reagents[chem])
+	. = ..()
+
+/decl/recipe/messatea
+	appliance = SAUCEPAN | POT
+	fruit = list("mtear" = 1)
+	reagents = list(/decl/reagent/water = 10)
+	resulting_reagents = list(/decl/reagent/messatea)
```
Comment (Arrow768) [inline code/modules/cooking/recipes/recipes_chems.dm:15]: That looks like a undocumented/unintentional change

**Review by Arrow768 — DISMISSED** [review]

File: code/modules/cooking/recipes/recipes_chems.dm (hunk 1 above)
Comment (MarinaGryphon) [reply code/modules/cooking/recipes/recipes_chems.dm:15]: OH NO

**Review by Geevies — APPROVED** [review]

**Pull request merged**